windows-driver-docs-pr/network/mb-modem-logging-with-dss.md

```diff
@@ -31,6 +32,8 @@ This new logging interface's design uses the control channel to set the logging
 3. Once it receives a success status code, the OS prepares to receive fragments from the modem. These fragments are called DataServiceSessionRead packets.
 4. DataServiceSessionRead packets continue to arrive until the OS issues another MBIM_CID_DSS_CONNECT command with the same DSS session ID and an MBIMDSSLinkDeactivate state.
 
+Once the modem writes any logs to the newly created data channel, the modem calls [**MbbDeviceReceiveDeviceServiceSessionData**](/windows-hardware/drivers/ddi/mbbcx/nf-mbbcx-mbbdevicereceivedeviceservicesessiondata), the data from which is available to apps via the WinRT layer : [**MobileBroadbandDeviceService**](/uwp/api/windows.networking.networkoperators.mobilebroadbanddeviceservice).
```

Comment:
In an internal discussion, it was decided that the modem logs need to be printable string data (such as an array of characters which can be redirected to the ETW session if needed). Please specify in this document that the modem logs need to be in format "printable string data which can be redirected to ETW session".